src/ralph/security/management/commands/update_is_patched.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import logging
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+
+from ralph.security.models import SecurityScan
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Update `is_patched` field on each SecurityScan'
+
+    def _update_is_patched(self):
+        # mark all as patched
+        patched = SecurityScan.objects.update(is_patched=True)
```

Comment:
IMO it'd be better to not do this, and instead:
* fetch all not_patched_ids (like below)
* filter sec. scan by these ids and only the ones which are patched (`is_patched=True`)
* update them to be not patched

(theoretically transactions should protect us before messing up here, but it'd be more db-friendly - less queries and, whats more important, less modifications (updates))